src/scripts/gym/GymList.ts

```diff
@@ -114,7 +117,7 @@ GymList['Viridian City'] = new Gym(
     6000,
     'Ha! That was a truly intense fight! You have won! As proof, here is the Earth Badge!',
     [
-        new GymBadgeRequirement(BadgeEnums.Volcano),
+        new QuestLineCompletedRequirement('Bill\'s Errand'),
```

Comment:
Does this need to be locked behind Sevii?

Reply:
Yes the intent is that this Sevii map becomes a required part of the Kanto experience. Which is also why I think it's a good place to put the client content.

The other Sevii maps, once added, should be fully optional for players that don't care about getting all the forms.

Reply:
It's also reflected in the NPC dialogue during the questline:
"I can see that your friend is eager to get back to Kanto and challenge the Pokémon League."